panic: invalid Go type int for field go_etcd_io.etcd.etcdserver.etcdserverpb.loggablePutRequest.value_size

Originally found by @bart0sh in kubernetes logs: https://github.com/kubernetes/test-infra/pull/17743#issuecomment-638107187

```
panic: invalid Go type int for field go_etcd_io.etcd.etcdserver.etcdserverpb.loggablePutRequest.value_size

goroutine 1 [running]:
google.golang.org/protobuf/internal/impl.newSingularConverter(0x177ec20, 0x15b9580, 0x177eb00, 0xc0002a55e0, 0x1777201, 0xc000128fa0)
        /Users/dims/go/pkg/mod/google.golang.org/protobuf@v1.23.0/internal/impl/convert.go:143 +0xff5
google.golang.org/protobuf/internal/impl.NewConverter(0x177ec20, 0x15b9580, 0x177eb00, 0xc0002a55e0, 0x203000, 0xc0001e1470)
        /Users/dims/go/pkg/mod/google.golang.org/protobuf@v1.23.0/internal/impl/convert.go:60 +0xd9
google.golang.org/protobuf/internal/impl.fieldInfoForScalar(0x177eb00, 0xc0002a55e0, 0x15b1656, 0x9, 0x0, 0x0, 0x177ec20, 0x15b9580, 0x15b1661, 0x2e, ...)
        /Users/dims/go/pkg/mod/google.golang.org/protobuf@v1.23.0/internal/impl/message_reflect_field.go:234 +0x110
google.golang.org/protobuf/internal/impl.(*MessageInfo).makeKnownFieldsFunc(0xc0002cc640, 0xffffffffffffffff, 0xffffffffffffffff, 0xffffffffffffffff, 0xffffffffffffffff, 0xc0001e1380, 0xc0001e13b0, 0xc0001e13e0, 0xc0001e1410)
        /Users/dims/go/pkg/mod/google.golang.org/protobuf@v1.23.0/internal/impl/message_reflect.go:67 +0x938
google.golang.org/protobuf/internal/impl.(*MessageInfo).makeReflectFuncs(0xc0002cc640, 0x177ec20, 0x165aaa0, 0xffffffffffffffff, 0xffffffffffffffff, 0xffffffffffffffff, 0xffffffffffffffff, 0xc0001e1380, 0xc0001e13b0, 0xc0001e13e0, ...)
        /Users/dims/go/pkg/mod/google.golang.org/protobuf@v1.23.0/internal/impl/message_reflect.go:36 +0x63
google.golang.org/protobuf/internal/impl.(*MessageInfo).initOnce(0xc0002cc640)
        /Users/dims/go/pkg/mod/google.golang.org/protobuf@v1.23.0/internal/impl/message.go:90 +0x182
google.golang.org/protobuf/internal/impl.(*MessageInfo).init(0xc0002cc640)
        /Users/dims/go/pkg/mod/google.golang.org/protobuf@v1.23.0/internal/impl/message.go:72 +0x3c
google.golang.org/protobuf/internal/impl.(*messageReflectWrapper).Has(0xc0002ca560, 0x177eb00, 0xc0002a5500, 0xc0002a5500)
        /Users/dims/go/pkg/mod/google.golang.org/protobuf@v1.23.0/internal/impl/message_reflect_gen.go:185 +0x33
github.com/golang/protobuf/proto.(*textWriter).writeMessage(0xc0001e1350, 0x177b660, 0xc0002ca560, 0x0, 0x0)
        /Users/dims/go/pkg/mod/github.com/golang/protobuf@v1.4.2/proto/text_encode.go:278 +0x925
github.com/golang/protobuf/proto.(*TextMarshaler).marshal(0x1b4c43c, 0x1772aa0, 0xc0001e12f0, 0x169d650, 0x9, 0x1b6c800, 0xf58, 0xf58)
        /Users/dims/go/pkg/mod/github.com/golang/protobuf@v1.4.2/proto/text_encode.go:86 +0x18a
github.com/golang/protobuf/proto.(*TextMarshaler).Text(...)
        /Users/dims/go/pkg/mod/github.com/golang/protobuf@v1.4.2/proto/text_encode.go:44
github.com/golang/protobuf/proto.CompactTextString(...)
        /Users/dims/go/pkg/mod/github.com/golang/protobuf@v1.4.2/proto/text_encode.go:106
go.etcd.io/etcd/etcdserver/etcdserverpb.(*loggablePutRequest).String(...)
        /Users/dims/go/pkg/mod/go.etcd.io/etcd@v0.5.0-alpha.5.0.20200401174654-e694b7bb0875/etcdserver/etcdserverpb/raft_internal_stringer.go:182
main.main()
        /Users/dims/go/src/k8s.io/kubernetes/main.go:9 +0xe0

Process finished with exit code 2
```

Easy to reproduce with the following snippet
```
package main

import (
	pb "go.etcd.io/etcd/etcdserver/etcdserverpb"
)

func main() {
	var req pb.PutRequest
	pb.NewLoggablePutRequest(&req).String()
}
```

--- a/etcdserver/etcdserverpb/raft_internal_stringer.go
+++ b/etcdserver/etcdserverpb/raft_internal_stringer.go
@@ -137,7 +137,7 @@ type loggableValueCompare struct {
 	Result    Compare_CompareResult `protobuf:"varint,1,opt,name=result,proto3,enum=etcdserverpb.Compare_CompareResult"`
 	Target    Compare_CompareTarget `protobuf:"varint,2,opt,name=target,proto3,enum=etcdserverpb.Compare_CompareTarget"`
 	Key       []byte                `protobuf:"bytes,3,opt,name=key,proto3"`
-	ValueSize int                   `protobuf:"bytes,7,opt,name=value_size,proto3"`
+	ValueSize int64                 `protobuf:"varint,7,opt,name=value_size,proto3"`
 	RangeEnd  []byte                `protobuf:"bytes,64,opt,name=range_end,proto3"`
 }
 
@@ -146,7 +146,7 @@ func newLoggableValueCompare(c *Compare, cv *Compare_Value) *loggableValueCompar
 		c.Result,
 		c.Target,
 		c.Key,
-		len(cv.Value),
+		int64(len(cv.Value)),
 		c.RangeEnd,
 	}
 }
@@ -160,7 +160,7 @@ func (*loggableValueCompare) ProtoMessage()    {}
 // To preserve proto encoding of the key bytes, a faked out proto type is used here.
 type loggablePutRequest struct {
 	Key         []byte `protobuf:"bytes,1,opt,name=key,proto3"`
-	ValueSize   int    `protobuf:"varint,2,opt,name=value_size,proto3"`
+	ValueSize   int64  `protobuf:"varint,2,opt,name=value_size,proto3"`
 	Lease       int64  `protobuf:"varint,3,opt,name=lease,proto3"`
 	PrevKv      bool   `protobuf:"varint,4,opt,name=prev_kv,proto3"`
 	IgnoreValue bool   `protobuf:"varint,5,opt,name=ignore_value,proto3"`
@@ -170,7 +170,7 @@ type loggablePutRequest struct {
 func NewLoggablePutRequest(request *PutRequest) *loggablePutRequest {
 	return &loggablePutRequest{
 		request.Key,
-		len(request.Value),
+		int64(len(request.Value)),
 		request.Lease,
 		request.PrevKv,
 		request.IgnoreValue,

